homeassistant/helpers/recorder/models.py

```diff
@@ -0,0 +1,13 @@
+"""Recorder models."""
```

Comment:
Do we need to break this up in so many files or can we put this just in `helpers/recorder/__init__.py`?

Reply:
We can put it all in one file

Reply:
f10b7493670b11ce28bace3921d9ca31bbe478b6